kogito-codegen/src/main/java/org/kie/kogito/codegen/process/ProcessCodegen.java

```diff
@@ -70,6 +73,7 @@
 
     private static final Logger LOGGER = LoggerFactory.getLogger(ProcessCodegen.class);
 
+    public static final Set<String> SUPPORTED_BPMN_EXTENSIONS = Collections.unmodifiableSet(new HashSet<>(Arrays.asList(".bpmn", ".bpmn2", ".bpmn-cm")));
```

Comment:
Could it be simpler like this:
```
public static final String[] SUPPORTED_BPMN_EXTENSIONS = {".bpmn", ".bpmn2", ".bpmn-cm"};
// filter(p -> Arrays.stream(SUPPORTED_BPMN_EXTENSIONS).anyMatch(p::endsWith))
```
Or, fancier like this:
```
    public enum SupportedBpmnExtensions {
     
        BPMN(".bpmn"), 
        BPMN2(".bpmn2"), 
        BPMN-CM(".bpmn-cm"); 
         
        private String extension;
     
        SupportedBpmnExtensions(String extension) {
            this.extension = extension;
        }
         
        public String getExtension() {
            return this.extension;
        }
     
        public static Stream<String> stream() {
            return Stream.Of(SupportedBpmnExtensions.values()).map(v -> v.getExtension)); 
        }
    }

```

Reply:
I prefer to keep the unmodifiableSet. Although the creation is a bit complex, the result grants an object that should not be modified.
Given we're just iterating over the elements I don't see the point in having an enum. If we were using the extensions in the code for making decisions, then yes I'd agree.

Thanks!